cmd/oci-create-runtime-bundle/main.go

```diff
@@ -112,29 +102,26 @@ func (v *bundleCmd) Run(cmd *cobra.Command, args []string) {
 	logrus.SetLevel(logLevel)
 
 	if v.version {
-		v.stdout.Printf("commit: %s", gitCommit)
-		v.stdout.Printf("spec: %s", specs.Version)
+		logrus.Infof("commit: %s", gitCommit)
+		logrus.Infof("spec: %s", specs.Version)
```

Comment:
`fmt.Printf` works for me.  It means that there's no way to capture the stream generated by the command (which you can currently do by setting `v.stdout` to something other than `os.Stdout`), but this is in `cmd/` and not in a library, so I think that decreased flexibility is fine.